community/modules/scheduler/schedmd-slurm-gcp-v6-controller/modules/slurm_files/scripts/tests/test_topology.py

```diff
@@ -89,3 +91,20 @@ def tpu_se(ns: TstNodeset) -> TstTPU:
     conf.gen_topology_conf(util.Lookup(cfg))
     want_written = PRELUDE + "\n".join(want_compressed) + "\n\n"
     assert open(cfg.output_dir + "/cloud_topology.conf").read() == want_written
+
```

Comment:
Suggest another test here demonstrating sorting of something like slurm-[10,5,12,7,22].